com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderCompilers.java

```diff
@@ -199,7 +208,24 @@ public ShaderProgramBuilder.ShaderCompileResult compile(ArrayList<ShaderCompileP
         }
     }
 
-    public static IShaderCompiler getCommonShaderCompiler(Platform platform) {
+    public static IShaderCompiler GetCommonShaderCompiler(Platform platform) {
         return new CommonShaderCompiler(platform);
     }
+
+    public static ArrayList<ShaderDesc.Language> GetSupportedOpenGLVersionsForPlatform(Platform platform) {
```

Comment:
Maybe we should use something like a static feature table (e.g like Platform.java has) for these things instead of multiple functions.. For now it's not a priority, but this could be data driven I think.